module/VuFind/src/VuFind/ILS/Driver/Voyager.php

```diff
@@ -891,16 +891,16 @@ protected function getMFHDData($record, $fieldSpecs)
             $subfieldCodes = substr($fieldSpec, 3);
             if ($fields = $record->getFields($fieldCode)) {
                 foreach ($fields as $field) {
-                    if ($subfields = $field->getSubfields()) {
+                    if ($subfields = $field['subfields'] ?? []) {
                         $line = '';
-                        foreach ($subfields as $code => $subfield) {
-                            if (false === strpos($subfieldCodes, (string)$code)) {
+                        foreach ($subfields as $subfield) {
+                            if (!strstr($subfieldCodes, $subfield['code'])) {
```

Comment:
Is there a strong reason to switch to using strstr here? Isn't strpos more efficient and less prone to error (e.g. if the subfield being evaluated somehow happens to be a "falsy" string)?

Reply:
That was copied by mistake from some local code. Reverted.